deploy/rbac/role.yaml

```diff
@@ -26,6 +26,22 @@ rules:
   verbs:
   - get
   - list
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
+- apiGroups:
+  - ""
+  resourceNames:
+  - policy-encryption-key
```

Comment:
Did you check the controller can still work with this restriction if there are additional secret in the same namespace?

Reply:
With `SelectorsByObject` then this is no longer a concern.